tests/generation/test_utils.py

```diff
@@ -1750,6 +1750,54 @@ def test_past_key_values_format(self):
                         past_kv[i][1].shape, (batch_size, num_attention_heads, seq_length, per_head_embed_dim)
                     )
 
+    def test_generate_from_inputs_embeds_decoder_only(self):
+        # When supported, tests that the decoder model can generate from `inputs_embeds` instead of `input_ids`
+        for model_class in self.all_generative_model_classes:
+            config, input_ids, _, _ = self._get_input_ids_and_config()
+
+            # Ignore:
+            # a) pad/eos effects (can't detect them after embedding),
```

Comment:
So if pad/eos exist (which is the most case), the result is not guaranteed to be the same, right?